### PR TITLE
feat(ui): Add user requests page

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

Adds a request list view at `/users/[userId]/requests` that only displays one user's requests.  The "Recent Requests" slider heading links to this page.

#### Screenshot (if UI related)

image
image

#### Todos

- [x] Successful build
- [x] Translation keys

### CODE REVIEW

File: server/routes/request.ts [hunk 1]
```diff
@@ -17,6 +17,9 @@ requestRoutes.get('/', async (req, res, next) => {
   try {
     const pageSize = req.query.take ? Number(req.query.take) : 10;
     const skip = req.query.skip ? Number(req.query.skip) : 0;
+    const requestedBy = req.query.requestedBy
+      ? Number(req.query.requestedBy)
+      : 0;
```
Comment (sct) [inline server/routes/request.ts:22]: Any reason we don't just set it to null here if there isn't a query value?

Comment (TheCatLady) [reply server/routes/request.ts:22]: Will change that! 👍🏻

File: src/pages/users/[userId]/requests.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { NextPage } from 'next';
+import RequestList from '../../../components/RequestList';
+import useRouteGuard from '../../../hooks/useRouteGuard';
+import { Permission } from '../../../hooks/useUser';
+
+const UserRequestsPage: NextPage = () => {
+  useRouteGuard(Permission.MANAGE_REQUESTS);
```
Comment (sct) [inline src/pages/users/[userId]/requests.tsx:8]: This guard will prevent people who can view requests from access this also?

Comment (TheCatLady) [reply src/pages/users/[userId]/requests.tsx:8]: Ah right, will add the View Requests perm as well.  Good catch, thanks 😄

**Review by sct — APPROVED** [review]

**Pull request merged**